its/src/test/java/com/sonar/it/csharp/AnalysisWarningsTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * SonarSource :: C# :: ITs :: Plugin
+ * Copyright (C) 2011-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package com.sonar.it.csharp;
+
+import com.sonar.it.shared.TestUtils;
+import com.sonar.it.shared.Tests;
+import com.sonar.orchestrator.build.BuildResult;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.sonarqube.ws.Ce;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static com.sonar.it.shared.Tests.ORCHESTRATOR;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AnalysisWarningsTest {
+  @Rule
+  public TemporaryFolder temp = TestUtils.createTempFolder();
+
+  @Before
+  public void init() {
+    TestUtils.reset(ORCHESTRATOR);
+  }
+
+  @Test
+  public void analysisWarningsImport() throws IOException {
+    Path projectDir = Tests.projectDir(temp, "Empty");
```

Comment:
Suggestion: Maybe I'd rename `Empty` to `EmptyProject` or even to `ProjectName` to be more explicit.
It was a bit tricky to understand if it was an empty directory of something else until I checked the `Tests.projectDir` definition.

Reply:
I'm not sure about this. `Empty` is one of the folders from the `projects` directory. None of them has the `Project`  suffix since the folder name already suggests that.

The name usually succinctly describes what it is about. Additionally, the method is called `projectDir` and it suggests that this is about a project.

`ProjectName` is too generic and does not describe what the project does.